### PR TITLE
Mat 6839/splitting the cql editor screen

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
Screenshot 2024-03-26 at 5 13 23 PM

## MADiE PR

Jira Ticket: [MAT-6839](https://jira.cms.gov/browse/MAT-6839)
(Optional) Related Tickets:

### Summary
splitting the cql editor screen

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/editor/MeasureEditor.tsx [hunk 1]
```diff
@@ -355,17 +361,61 @@ const MeasureEditor = () => {
...
+                <MadieEditor
+                  onChange={(val: string) => handleMadieEditorValue(val)}
+                  value={editorVal}
+                  inboundAnnotations={elmAnnotations}
+                  inboundErrorMarkers={errorMarkers}
+                  height="calc(100vh - 135px)"
+                  readOnly={!canEdit}
+                  setOutboundAnnotations={setOutboundAnnotations}
+                />
+                <div
```
Comment (adongare) [inline src/components/editMeasure/editor/MeasureEditor.tsx]: I see similar action row on line 433 as well. can we not reuse it?

Comment (sb-prateekkeerthi) [reply src/components/editMeasure/editor/MeasureEditor.tsx]: the stylings for the footer section is different when compared to the ones which we have when no right and left panel are involved, so I had to used the buttons section twice. when we implement the same functionality for QI Core at that time it would be just easy to remove that part of code without looking out for any conditions

Comment (adongare) [reply src/components/editMeasure/editor/MeasureEditor.tsx]: that's weird we have two different styles

Comment (sb-prateekkeerthi) [reply src/components/editMeasure/editor/MeasureEditor.tsx]: I confirmed with Michael about discrepencies in current editor and figma styles and I have changed the current styles to match the figma styles

**Review by adongare — APPROVED** [review]
